<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Btn_Right">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>45</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Down">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>45</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Up">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Enter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>ENTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Esc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_SendPic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>加载图片</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>140</y>
       <width>105</width>
       <height>10</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_Sta">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>125</y>
       <width>105</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>点击发送图片</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_backled">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>107</y>
       <width>160</width>
       <height>15</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_BackLed">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>105</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_LedOn">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>0</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LED ON</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Config:8-N-1;Baud:9600</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
    <widget class="QLabel" name="Lab_FaultCode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fault code: 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_FaultLevel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fault level: 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_HeartBeat">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Heart beat: 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_Boot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Boot status: Off</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_CurWin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current windows: win 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_CurPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current page: page 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_Softver">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Soft version: </string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_HardVer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hard version: </string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_BootSta">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>164</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>COM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_ComSta">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>164</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>BOOT:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>200</width>
      <height>52</height>
     </rect>
    </property>
    <widget class="QLabel" name="Lab_LedRGB">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>0</y>
       <width>25</width>
       <height>52</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_R">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>0</y>
       <width>40</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_G">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>17</y>
       <width>40</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>G 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_B">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>37</y>
       <width>40</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B 0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_B">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>39</y>
       <width>130</width>
       <height>12</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_G">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>19</y>
       <width>130</width>
       <height>12</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_R">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>130</width>
       <height>12</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Lab_fgx">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>0</y>
      <width>2</width>
      <height>235</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>185</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>16</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Author : LH</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>210</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Virtual Keyboard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Version : V1.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
